qrenderdoc/Code/QRDUtils.cpp

```diff
@@ -1653,6 +1653,7 @@ QString D3DSemanticString(const SigParameter &sig)
       lit("SV_Barycentrics"),
       lit("SV_CullPrimitive"),
       lit("out indices"),
+      lit("Unsupported (multiview index)"),
```

Comment:
This is available as `SV_ViewID` in D3D12 at least. This list doesn't make a distinction for D3D11 so it can just be used unconditionally.

(side note, I see some earlier instances of features that D3D12 has gained in at least one case so this list might need re-examining at some point. The generated edit stubs are not used much these days though with how complex shaders often are and how easily available original source editing is).